### PR TITLE
[INCR-92] Run python-modernize on RecommenderXBlock

### PR DESCRIPTION
**Pull request by cclauss**
Fixes: https://openedx.atlassian.net/browse/INCR-92

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @cclauss! I've created [OSPR-3121](https://openedx.atlassian.net/browse/OSPR-3121) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

File: recommender/recommender.py [hunk 1]
```diff
@@ -12,15 +13,16 @@
 from copy import deepcopy
 
 from mako.lookup import TemplateLookup
-from urllib import unquote_plus
-from urlparse import urlparse, urlunparse
+from six.moves.urllib.parse import unquote_plus
+from six.moves.urllib.parse import urlparse, urlunparse
```
Comment (jmbowman) [inline recommender/recommender.py]: These lines can be combined now that they're importing from the same package.

File: requirements.txt [hunk 2]
```diff
@@ -3,3 +3,5 @@ fs
 mako
 boto
 bleach==2.1.4
+pycodestyle
+pylint
```
Comment (jmbowman) [inline requirements.txt]: Probably not worth adding these until we actually run them, and once we do we'd want to use edx-lint instead of pylint directly.

Comment (cclauss) [reply requirements.txt]: Added a run of __pycodestyle__ but removed __pylint__.

Do we acutally _run_ __edx-lint__ somewhere?  I see it in requirements files but I do not see where it is actually run.

**Comment by edx-webhook** [comment]
@cclauss 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/RecommenderXBlock&entry.867055334=https://github.com/edx/RecommenderXBlock/pull/10&entry.1484655318=https://github.com/cclauss&entry.752974735=2019-02-22+23:01&entry.1917517419=2019-02-23+21:34&entry.2133058324=Yes) so we can improve your experience in the future.

**Comment by natabene** [comment]
@cclauss Thanks for your contribution.

**Pull request merged**